cmd/kubeadm/app/node/csr.go

```diff
@@ -36,17 +35,15 @@ import (
 
 // PerformTLSBootstrap creates a RESTful client in order to execute certificate signing request.
 func PerformTLSBootstrap(s *kubeadmapi.KubeadmConfig, apiEndpoint string, caCert []byte) (*clientcmdapi.Config, error) {
-	// TODO(phase1+) try all the api servers until we find one that works
+	// TODO(phase2) try all the api servers until we find one that works
 	bareClientConfig := kubeadmutil.CreateBasicClientConfig("kubernetes", apiEndpoint, caCert)
 
 	hostName, err := os.Hostname()
 	if err != nil {
 		return nil, fmt.Errorf("<node/csr> failed to get node hostname [%v]", err)
 	}
 
-	// TODO: hostname == nodename doesn't hold on all clouds (AWS).
-	// But we don't have a cloudprovider, so we're stuck.
-	glog.Errorf("assuming that hostname is the same as NodeName")
```

Comment:
We don't use glog (or at least not yet), as it produces very noisy output. This message wasn't quite user-friendly also, we should prioritise https://github.com/kubernetes/kubernetes/issues/33641 instead.